Git-commit: 8698fbee790d4c6d5233383e14613105d9bf3c17
From: Xie XiuQi <xiexiuqi@huawei.com>
Date: Fri, 26 Feb 2021 20:21:20 +0800
Subject: [PATCH] resctrlfs: fix up RESCTRL dependency
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

hulk inclusion
category: bugfix
feature: ARM MPAM support
bugzilla: 48265
CVE: NA

--------------------------------

On x86 platform,
make allmodconfig && make -j64

  CALL    scripts/checksyscalls.sh
  DESCEND  objtool
  CHK     include/generated/compile.h
  CC      fs/resctrlfs.o
fs/resctrlfs.c:36:25: fatal error: asm/resctrl.h: No such file or
directory
 #include <asm/resctrl.h>
                         ^
compilation terminated.
make[1]: *** [fs/resctrlfs.o] Error 1
make: *** [fs] Error 2

Make RESCTRL do not depend on INTEL_RDT.

Signed-off-by: Xie XiuQi <xiexiuqi@huawei.com>
Reviewed-by: Yang Yingliang <yangyingliang@huawei.com>
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Cheng Jian <cj.chengjian@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 fs/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/Kconfig b/fs/Kconfig
index ba8e9ad74f24..b21b7213f521 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -354,7 +354,7 @@ endmenu
 config RESCTRL
 	bool "Support Memory Partitioning and Monitoring"
 	default n
-	depends on INTEL_RDT || MPAM
+	depends on MPAM
 	select KERNFS
 	help
 	  Memory Partitioning and Monitoring. More exactly Memory system
-- 
2.31.1

